mitiq/zne/tests/test_zne.py

```diff
@@ -630,3 +630,50 @@ def executor(circuit):
         scale_noise=noise_scaling_method,
     )
     assert np.isclose(zne_res, two_stage_zne_res)
+
+
+@pytest.mark.parametrize(
+    "extrapolation_factory", [RichardsonFactory, LinearFactory]
+)
+@pytest.mark.parametrize(
+    "to_frontend",
+    [None, to_qiskit, to_braket, to_pennylane, to_pyquil, to_qibo],
+)
+def test_two_stage_zne_default_scaling(extrapolation_factory, to_frontend):
+    qreg = cirq.LineQubit.range(2)
+    cirq_circuit = cirq.Circuit(
+        cirq.H.on_each(qreg),
+        cirq.CNOT(*qreg),
+        cirq.CNOT(*qreg),
+        cirq.H.on_each(qreg),
+    )
+    if to_frontend is not None:
+        frontend_circuit = to_frontend(cirq_circuit)
+    else:
+        frontend_circuit = cirq_circuit
+
+    scale_factors = [1, 3, 5]
+    circs = scaled_circuits(frontend_circuit, scale_factors)
+
+    assert len(circs) == len(scale_factors)
+
+    np.random.seed(42)
+
+    def executor(circuit):
+        return np.random.random()
+
+    results = [executor(cirq_circuit) for _ in range(3)]
+    extrapolation_method = extrapolation_factory.extrapolate
+    two_stage_zne_res = combine_results(
+        scale_factors, results, extrapolation_method
+    )
+
+    assert isinstance(two_stage_zne_res, float)
+
+    np.random.seed(42)
+    zne_res = execute_with_zne(
+        cirq_circuit,
+        executor,
+        factory=extrapolation_factory(scale_factors),
+    )
+    assert np.isclose(zne_res, two_stage_zne_res)
```

Comment:
These last two tests (assertions) seem unrelated to the code change in `zne.py`.

1. The first tests that the circuits generated by `fold_gates_at_random` return floats. This should be covered by tests in `test_folding.py`.
2. The second tests the performance of a specific ZNE configuration, which I think we want to avoid as it creates non-deterministic tests.

Reply:
yeah, I copy-pasted the existing test to understand if I needed to have any newer unit tests. 

Sorry, this PR is not ready yet!